neo/framework/Common.cpp

```diff
@@ -1291,7 +1313,7 @@ Com_EditPDAs_f
 ==================
 */
 static void Com_EditPDAs_f( const idCmdArgs &args ) {
```

Comment:
This should not be hidden behind ID_ALLOW_TOOLS (which is specific to the MFC-tools) anymore, similar to my changes for the light editor.
I think it makes most sense to just always provide that command, even if in (the rare) case that ImGui is disabled it only prints a message that it's not available.

similarly below for `cmdSystem->ADdCommand( "editPDAs", ...)`

Reply:
keep in mind that the `#ifndef IMGUI_DISABLE` is needed in here so it still works when not building with ImGui, which especially is the case when building dhewm3ded (the dedicated server)